dashboards/ebs/ebs_snapshot_detail.sp

```diff
@@ -37,7 +37,7 @@ dashboard "aws_ebs_snapshot_detail" {
     graph {
       type  = "graph"
       title = "Relationships"
-      query = query.aws_ebs_snapshot_relationships
+      query = query.aws_ec2_ebs_relationships_graph
```

Comment:
The old query name looked ok to me, any reason we changed it and also included `ec2` (especially since this is for EBS snapshots)?

Reply:
I think this should be `aws_ebs_snapshot_relationships_graph`